src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -4325,11 +4325,20 @@ export interface InfiltrationReward {
  * @public
  */
 export interface InfiltrationLocation {
-  location: any;
+  location: SimpleLocation;
```

Comment:
This looks like it should be `Location` instead based on the return signature from https://github.com/danielyxie/bitburner/blob/e85a66202a74f7bac458d4bb4438ad246f6aa017/src/NetscriptFunctions/Infiltration.ts#L19-L38
```suggestion
  location: Location;
```